lib/node/Status.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const clone = require('../util/clone');
+
```

Comment:
don't need new line here I think? or do you want to sort them by directory they're coming from? or by if they start with a capital letter? let me know what you were thinking :)